pkg/apis/policy/types.go

```diff
@@ -182,6 +182,10 @@ type PodSecurityPolicySpec struct {
 	SELinux SELinuxStrategyOptions
 	// RunAsUser is the strategy that will dictate the allowable RunAsUser values that may be set.
 	RunAsUser RunAsUserStrategyOptions
+	// RunAsGroup is the strategy that will dictate the allowable RunAsGroup values that may be set.
+	// If this field is omitted, RunAsGroup can take any value. This field requires the RunAsGroup
```

Comment:
nit: clarify "If this field is omitted, **the pod's** RunAsGroup can take any value."

Reply:
Also need to update the comment in staging/src/k8s.io/api/policy/v1beta1/types.go and (maybe) staging/src/k8s.io/api/extensions/v1beta1/types.go